component/vshn_postgres.jsonnet

```diff
@@ -204,6 +204,10 @@ local sgInstanceProfile = {
                 spec: {
                   cpu: '',
                   memory: '',
+                  requests: {
+                    cpu: null,
+                    memory: null,
+                  },
```

Comment:
Did you double check that this doesn't create a diff in ArgoCD, if no requests are set?

Reply:
Yes, there is no diff. However, the manifest on the cluster is `requests: {}`.
I would still like to keep it thought for readability purposes as we do patch the `cpu` and `memory` fields in the composition.